protos/protos/public/modeldb/versioning/VersioningService.proto

```diff
@@ -418,6 +418,40 @@ message RevertRepositoryCommitsRequest {
     }
 }
 
+message FindRepositories {
+    repeated RepositoryIdentification repository_id = 1;
+    repeated KeyValueQuery predicates = 2;
+    bool ids_only = 3; // whether to return full experiment runs or ids only
+
+    //For pagination
+    int32 page_number = 4;
+    int32 page_limit = 5;
+    bool ascending = 6;
+    string sort_key = 7;
```

Comment:
add workspace_name as per the suggestion for endpoint

Reply:
Added workspace_name